custom/icds_reports/ucr/data_sources/ccs_record_cases_monthly_tableau.json

```diff
@@ -1982,41 +1981,35 @@
         "operator": "gt",
         "property_value": 0,
         "expression": {
-          "type": "reduce_items",
-          "aggregation_fn": "count",
-          "items_expression": {
-            "type": "icds_get_case_forms_by_date",
-            "xmlns": [
-              "http://openrosa.org/formdesigner/2864010F-B1B1-4711-8C59-D5B2B81D65DB"
-            ],
-            "start_date": {
-              "type": "icds_month_start"
-            },
-            "end_date": {
-              "type": "icds_month_end"
-            }
-          }
+          "type": "icds_get_case_forms_in_date",
```

Comment:
This needs a `case_id_expression`. Can either add it here or make it not required in ucr/expressions